bundles/binding/org.openhab.binding.xbmc/src/main/java/org/openhab/binding/xbmc/rpc/XbmcConnector.java

```diff
@@ -581,25 +586,82 @@ private void requestPlayerUpdate(int playerId, boolean updatePolledPropertiesOnl
-                    // now update each of the openHAB items for each property
-                    for (String property : properties) {
-                        String value = item.getPropertyValue(property);
-                        if (property.equals("Player.Fanart")) {
-                            updateFanartUrl(property, value);
-                        } else {
+            final List<String> propertiesInfo = new ArrayList<String>();
+            final List<String> propertiesProperties = new ArrayList<String>();
+            final List<String> propertiesLabels = new ArrayList<String>();
+
+            for (String property : properties) {
+                if (property.startsWith("Player.")) {
+                    propertiesInfo.add(property);
+                }
+                if (property.startsWith("Property.")) {
+                    propertiesProperties.add(property);
+                }
+                if (property.startsWith("Label.")) {
+                    propertiesLabels.add(property);
+                }
+            }
+
+            // make the request for the player item details using GETINFO
+            if (!propertiesInfo.isEmpty()) {
+                final PlayerGetItem item = new PlayerGetItem(client, httpUri);
+                item.setPlayerId(playerId);
+                item.setProperties(propertiesInfo);
+
+                item.execute(new Runnable() {
+                    @Override
+                    public void run() {
+                        // now update each of the openHAB items for each property
+                        for (String property : propertiesInfo) {
+                            String value = item.getPropertyValue(property);
+                            // logger.debug("UPDATEfromGETINFO: property=" + property + " & value=" + value);
+                            if (property.equals("Player.Fanart")) {
+                                updateFanartUrl(property, value);
+                            } else {
+                                updateProperty(property, value);
+                            }
+                        }
+                    }
+                });
+            }
+
+            // make the request for the player item2 details using GETPROPERTIES
+            if (!propertiesProperties.isEmpty()) {
+                final PlayerGetProperties item2 = new PlayerGetProperties(client, httpUri);
+                item2.setPlayerId(playerId);
+                item2.setProperties(propertiesProperties);
+
+                item2.execute(new Runnable() {
+                    @Override
+                    public void run() {
+                        // now update each of the openHAB items for each property
+                        for (String property : propertiesProperties) {
+                            String value = item2.getPropertyValue(property);
+                            // logger.debug("UPDATEfromGETPROPERTIES: property=" + property + " & value=" + value);
```

Comment:
Either remove the commented out code, or uncomment the debug line (and switch to parameterized logging).